guides/source/action_mailbox_basics.md

```diff
@@ -277,13 +294,62 @@ class ApplicationMailbox < ActionMailbox::Base
 end
 ```
 
-Then set up a mailbox:
+The regular expression matches the incoming email's `to`, `cc`, or `bcc` fields. For example, the above will match any email sent to `save@` to a 'forwards' mailbox. There are other ways to route an email, see [`ActionMailbox::Base API`](https://api.rubyonrails.org/classes/ActionMailbox/Base.html) for more.
+
+We need to create that 'forwards' mailbox next.
+
+### Create a Mailbox
 
 ```bash
 # Generate new mailbox
 $ bin/rails generate mailbox forwards
 ```
 
+This creates `app/mailboxes/forwards_mailbox.rb`, with a `ForwardsMailbox` class and a `process` method.
+
+### Process Email
+
+When processing an `InboundEmail`, you can get the parsed version of the email as a [`Mail`](https://github.com/mikel/mail) object with `InboundEmail#mail`. You can also get the raw source directly using the `#source` method. With `Mail` object, you can access the relevant fields, such as `mail.to`, `mail.body.decoded`, etc.
+
+```bash
```

Comment:
I love seeing this code walkthrough explicitly, very helpful.

Reply:
The README of that `mail` library was good read :)